docs/api/sql/Raster-operators.md

```diff
@@ -521,6 +521,42 @@ SELECT RS_BandPixelType(RS_MakeEmptyRaster(2, "I", 5, 5, 53, 51, 1, 1, 0, 0, 0),
 
 Output: `IllegalArgumentException: Provided band index 3 is not present in the raster`
 
+### RS_Count
+
+Introduction: Returns the number of pixels in a given band. If band is not specified then it defaults to `1`.
+
+!!!Note
```

Comment:
Also add note and example explaining that this function can thrown an IllegalArgumentException